### ISSUE TITLE
Is script for exclusions updated?

### ISSUE THREAD

**Issue by Nigward8174** [UnamSanctam/UnamDownloader#43]
Is script for exclusions updated? You said that was the reason why the miner got detected

**Comment by Nigward8174** [comment]
Also it would be really nice to add in downloader a feature to download using rentry because if someone has like site where they uploaded it, they wouldn't have to replace the files, but just the link

**Comment by UnamSanctam** [comment]
> Is script for exclusions updated? You said that was the reason why the miner got detected

No, thought it shouldn't be statically detected at the moment because it's not the same full command as the miner. Though I am changing some things in the next update.

> Also it would be really nice to add in downloader a feature to download using rentry because if someone has like site where they uploaded it, they wouldn't have to replace the files, but just the link

Yes though it's really hard to add in a user friendly manner, it has to be as obvious as possible what the options mean. There could be something like two radio buttons or tabs that you switch between, one called "Direct Download" and one called something else. I just don't know what it would be, something like "Hosted Link" or something isn't really too obvious and I don't want to call it "Pastebin Link" like it is "normally" in most programs since then a lot of people think that you can only use Pastebin.

**Comment by Nigward8174** [comment]
Well if you name first one "Direct Download", something like "Third party download" would be fine

**Comment by Nigward8174** [comment]
But since 99% of people are downloading from Releases, they usually all read the changelog so they understand what every button does

**Comment by UnamSanctam** [comment]
Maybe, the name doesn't really make it seem obvious though, there are a lot of people (that just finds it randomly through Google) that don't read change logs or anything, and the change log that would say it would only be for the most recent release. Maybe "Direct Download" and "Pastebin Link Download" would be the most obvious, since pastebin is technically the "real" term for it.

**Comment by Nigward8174** [comment]
Well that's fine too

**Comment by Nigward8174** [comment]
Do you know an ETA when it should be updated @UnamSanctam
